src/controllers/auth.controller.ts

```diff
@@ -28,12 +34,12 @@ import {
 
         const accessToken = sign({
             id: user.id
-        }, "ACCESS_TOKEN",
+        }, config.accessTokenSecret as string ,
```

Comment:
It good practice to not use `as string`, instead add `!` at end of the config like this: `config.accessTokenSecret!`